codalab/model/bundle_model.py

```diff
@@ -2910,18 +2961,71 @@ def create_bundle_store(self, user, name, storage_type, storage_format, url, aut
             connection.execute(cl_bundle_store.insert().values(bundle_store_value))
         return uuid
 
-    def save_oauth2_auth_code(self, grant):
-        with self.engine.begin() as connection:
-            result = connection.execute(oauth2_auth_code.insert().values(grant.columns))
-            grant.id = result.lastrowid
-        return grant
-
-    def delete_oauth2_auth_code(self, auth_code_id):
+    def update_bundle_store(self, user, uuid, update_fields):
```

Comment:
Add type annotations to all new parameters. Can you also add it to create_bundle_store above?